composer.json

```diff
@@ -0,0 +1,73 @@
+{
+    "name": "laravel/laravel",
+    "type": "project",
+    "description": "The Laravel Framework.",
+    "keywords": ["framework", "laravel"],
```

Comment:
These fields should be renamed to something related with this project, I think so.